pgoapi/location.py

```diff
@@ -27,9 +27,28 @@ def get_route(start, end, use_google=False, GMAPS_API_KEY="", walk_to_all_forts=
         else:
             d = directions_service.directions(origin, destination, mode="walking", units="metric")
         steps = d[0]['legs'][0]['steps']
-        return [(step['end_location']["lat"], step['end_location']["lng"]) for step in steps]
+        final_steps = [
```

Comment:
I may chose to add waypoint feature in the future to use a local lib to solve the Traveling Salesman problem and visit all forts within vicinity (directly) which will be faster, no google api required and no "fort out of range" issue.